lib/rucio/richclient.py

```diff
@@ -0,0 +1,308 @@
+    the table. Strings with style markup will be rendered as styled text.
+
+    :param table_format: style of the table
+    :param headers: list of headers
+    :param rows: list of rows
+    :param col_alignments: list of column alignments
+    :param row_styles: list of row styles
+    :returns: a Rich Table object
+    """
+    table = Table(box=table_format, show_header=headers is not None and len(headers) > 0)
+    table.row_styles = row_styles or ['none', 'dim']
+
+    if len(rows) == 0:
+        if headers:
+            for header in headers:
+                table.add_column(header)
+        return table
+
+    # Auto-detect on first row, numerical values on the right.
+    col_alignments = col_alignments or ['right' if str(col).isnumeric() else 'left' for col in rows[0]]
+    headers = headers or [''] * len(rows[0])
+
+    if col_alignments and len(headers) > len(col_alignments):
+        while len(headers) > len(col_alignments):
+            col_alignments.append('left')
+
+    headers = headers or [''] * len(rows[0])
+    for i, header in enumerate(headers):
+        table.add_column(header, overflow='fold', justify=col_alignments[i])
+
+    for row in rows:
+        row = ['' if str(col) == 'None' else col for col in row]
+        table.add_row(*row)
+    return table
+
+
+def print_output(
+    *output: Any,
+    console: Console,
+    no_pager: bool = False
+) -> None:
+    """
+    Prints the objects using the specified Rich console object. Optionally disables the pager if specified.
+
+    The function works similarly to Rich's `console.print()` method but provides additional control over the pager feature.
+
+    :param output: objects to print to the terminal
+    :param console: Rich console object
+    :param no_pager: flag to disable the pager
+    """
+    if console.is_terminal:
+        if no_pager:
+            console.print(*output)
+        else:
+            console.width = sys.maxsize   # Overwrite auto-detected console width.
+            console.begin_capture()
+            console.print(*output)
+    else:
+        console.width = sys.maxsize
+        console.print(*output)
+
+
+def get_cli_config() -> str:
+    """
+    Returns the CLI type from the config file.
+
+    :returns: CLI type
+    """
+    try:
+        cli_type = config_get('experimental', 'cli').lower()
+        if cli_type not in ['rich', 'tabulate']:
+            cli_type = 'tabulate'
+    except (NoOptionError, NoSectionError, RucioException):
+        cli_type = 'tabulate'
+    return cli_type
+
+
+def get_pager() -> Callable[[str], None]:
+    """
+    Returns the pager function based on the terminal availability.
+
+    :returns: pager
+    """
+    default_pager = 'less'
+    try:
+        result = subprocess.run([default_pager], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        less_available = result.returncode == 0
+    except (FileNotFoundError, Exception):
```

Comment:
If you’re catching Exception, is there a benefit to mentioning `FileNotFoundError`?